WebDevStudios/wds-block-starter
# Feature/wp scripts 12.1.1

## WP Scripts Update
This update has some breaking changes that will change how we import and generate SCSS. Since WP Scripts now supports SCSS & PostCSS natively, we need to adjust the Webpack configuration and at import statements to the files as needed.

## Description
- Remove MiniCssExtractPlugin since this feature is being included in WP Scripts
- Remove style.scss & editor.scss glob entries in favor of WP Scripts default handling of style extracts
- Remove rules custom module rules since these features are included in WP Scripts core functionality
- Remove extra files from emit plugin that are no longer generated
- Update package to 12
- Generate updated lock file
- Add editor scss import to edit.js
- Add style scss import to index.js
- Bump version

### Steps to verify the solution
1. Checkout this branch locally and run `npm install`.
2. Verify that your files build as expected.
3. Check that the block runs without issues.

- [NA] Is this issue accessible? (Section 508/WCAG 2.1AA)
- [NA] Does this pass CrossBrowserTesting.com? (Edge, Safari, Chrome, Firefox)
- [YES] Will this pull request require [updating the wiki?](https://github.com/WebDevStudios/wds-block-starter/wiki)

Details to the Wiki will need updated after this update is approved.
